crypto/x509v3/punycode.c

```diff
@@ -0,0 +1,498 @@
+/*
+punycode.c from RFC 3492
```

Comment:
We seem to have two copies of this same code?

Reply:
Sure. It's a rebase artifact. Will fix.

Reply:
Fixed.